ops/services/app_functions/report_stream_batched_publisher/functions/QueueBatchedReportStreamUploader/index.ts

```diff
@@ -111,6 +113,14 @@ const QueueBatchedTestEventPublisher: AzureFunction = async function (
       },
       tagOverrides,
     });
+
+    if(postResult.status === 400) {
+      //publish messages to file failure queue
+      await publishToFileErrorQueue(context, fileErrorQueue, messages);
```

Comment:
if publishing to file error queue fails, can we skip deleting? like a try catch or something

Reply:
I guess I was assuming if it failed it would throw an exception, but I'll update this to be a little more certain.

Reply:
oh yes you're right that would make sense